src/dbus.c

```diff
@@ -200,6 +201,15 @@ static void on_notify(GDBusConnection * connection,
                                         if (dict_value)
                                                 raw_icon = get_raw_image_from_data_hint(dict_value);
 
+                                        dict_value = g_variant_lookup_value(content, "transient", G_VARIANT_TYPE_INT32);
```

Comment:
Thanks for the feedback. At first, I thought shame on me. 🙈

> The value type for the hint dictionary in D-BUS is of the DBUS_TYPE_VARIANT container type. This allows different data types (string, integer, boolean, etc.) to be used for hints.

So let's look at `notify-send`:

```
[bebe:~] % notify-send asdf 'asdf asdf asdf asdf asdf asdf asdf' --hint=bool:transient:1
Invalid hint type "bool". Valid types are int, double, string and byte.
```

It would be stupid to require this a boolean. We have to fix upstream bugs first, if we want to stick to the standard. 🤣

I have the feeling, that the gnome devs do not even look at their own standard sometimes.